src/app/application-form/preview/preview.component.ts

```diff
@@ -8,5 +8,18 @@ import { ApplicationFormValues } from '../application-form-values';
   styleUrls: ['./preview.component.scss'],
 })
 export class PreviewComponent {
-  constructor(@Inject(MAT_DIALOG_DATA) public data: ApplicationFormValues) {}
+  dob: string;
+  partner: string;
+
+  constructor(@Inject(MAT_DIALOG_DATA) public data: ApplicationFormValues) {
+    data.financialInformation.sourceOfIncome?.toUpperCase;
+    this.dob = data.customerInformation.dateOfBirth.toLocaleDateString('lt-LT');
+    if (data.financialInformation.coBorrower === 'true') {
```

Comment:
This seems like the perfect place for a ternary expression.
`this.partner = data.financialInformation.coBorrower === "true" ? "Yes" : "No"`